mentoring/mentoring.py

```diff
@@ -370,10 +446,14 @@ def handleAssessmentSubmit(self, submissions, suffix):
             'num_attempts': self.num_attempts,
             'step': self.step,
             'score': score.percentage,
-            'correct_answer': score.correct,
-            'incorrect_answer': score.incorrect,
-            'partially_correct_answer': score.partially_correct,
-            'message': message
+            'correct_answer': len(score.correct),
+            'incorrect_answer': len(score.incorrect),
+            'partially_correct_answer': len(score.partially_correct),
```

Comment:
@Kelketek probably these three context variables could be removed and `len(correct)` be used in template instead.